data.js

```diff
@@ -38,6 +38,14 @@ module.exports = [{
     url: "https://github.com/eorroe/NodeList.js",
     source: "https://raw.githubusercontent.com/eorroe/NodeList.js/master/NodeList.js"
   },
+  {
+    name: "Star Rating",
+    github: "geminilabs/star-rating.js",
+    tags: ["rating", "stars", "star-rating"],
+    description: "Transforms a SELECT element into a dynamic star rating control",
+    url: "https://geminilabs.github.io/star-rating.js/",
+    source: "https://raw.githubusercontent.com/geminilabs/star-rating.js/master/src/star-rating.js"
```

Comment:
Should be an array of paths for both the needed JS and CSS.

Reply:
Done